### PR TITLE
Androapp 4925

### PR DESCRIPTION
**Pull request by CarlosMacaneta**
## Description
Please include a summary of the change and include the related jira issue if it exists.

[ jira issue ](https://jira.dhis2.org/browse/

## Solution description
If this PR is a fix include a brief description on how the issue is solved.
## Covered unit test cases
Describe the tests that you ran to verify your changes.
## Where did you test this issue?
- [x] Smartphone Emulator
- [ ] Tablet Emulator
- [x] Smartphone
- [ ] Tablet
## Which Android versions did you test this issue?
- [ ] 4.4
- [ ] 5.X - 6.X
- [ ] 7.X
- [x] 8.X
- [ ] 9.X - 10.X
- [x] Other
## Checklist
- [x] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code

### CODE REVIEW

**Review by mvmmacamo2 — APPROVED** [review]

File: stock-managment/src/main/java/org/dhis2/android/rtsm/ui/home/screens/components/MainContent.kt [hunk 1]
```diff
@@ -187,7 +196,28 @@ fun MainContent(
         }
 
         Column(horizontalAlignment = Alignment.CenterHorizontally) {
-            Text(text = "Table content", textAlign = TextAlign.Center)
+            if (viewModel.toolbarTitle.collectAsState().value.name
+                == TransactionType.DISTRIBUTION.name
+            ) {
+                if (hasFacilitySelected && hasDestinationSelected == true) {
+                    MdcTheme {
```
Comment (Balcan) [inline stock-managment/src/main/java/org/dhis2/android/rtsm/ui/home/screens/components/MainContent.kt:203]: In compose, the theme has to be the top element in the tree.
`setContent {
  MdcTheme{
    MainContent()
  }
}
`

**Review by Balcan — CHANGES_REQUESTED** [review]

**Review by mvmmacamo2 — CHANGES_REQUESTED** [review]
Make proceed button work as before or make it work for current table

File: stock-managment/src/main/java/org/dhis2/android/rtsm/ui/home/HomeActivity.kt [hunk 2]
```diff
@@ -164,7 +167,7 @@ class HomeActivity : AppCompatActivity(), OnOrgUnitSelectionFinished {
             }
             return
         }
-        startActivity(
+        /*startActivity(
```
Comment (andresmr) [inline stock-managment/src/main/java/org/dhis2/android/rtsm/ui/home/HomeActivity.kt:170]: why is this commented?

File: stock-managment/src/main/java/org/dhis2/android/rtsm/ui/managestock/ManageStockViewModel.kt [hunk 3]
```diff
@@ -50,11 +50,22 @@ class ManageStockViewModel @Inject constructor(
     schedulerProvider,
     speechRecognitionManager
 ) {
-    val transaction: Transaction = savedState.get<Transaction>(INTENT_EXTRA_TRANSACTION)
-        ?: throw InitializationException("Transaction information is missing")
+    private val _config = MutableLiveData(
```
Comment (andresmr) [inline stock-managment/src/main/java/org/dhis2/android/rtsm/ui/managestock/ManageStockViewModel.kt:53]: I would not hardcore the values, could you send via parameters?

**Review by andresmr — CHANGES_REQUESTED** [review]

File: stock-managment/src/main/java/org/dhis2/android/rtsm/ui/home/HomeActivity.kt (hunk 2 above)
Comment (CarlosMacaneta) [reply stock-managment/src/main/java/org/dhis2/android/rtsm/ui/home/HomeActivity.kt:170]: resolved

**Review by andresmr — APPROVED** [review]

**Review by andresmr — APPROVED** [review]

**Review by mvmmacamo2 — APPROVED** [review]

**Pull request merged**